Gemfile

```diff
@@ -1,3 +1,6 @@
 source 'https://rubygems.org'
 
 gemspec
+
+gem "pry"
+gem "pry-byebug"
```

Comment:
Ok I wrapped them in a group. I didn't put them in the gemspec I guess because they're not really essential dependencies... you can run the specs without them, but I like to have them around. Seems like in that case people often put them in the `Gemfile` and not in the gemspec.